### ISSUE TITLE
Implement field-specific encoding in custom format

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/export_service#99]
ADS Classic implements an option to specify an encoding on a field by field basis.  As specified in the ADS Classic help page (http://adsabs.harvard.edu/abs_doc/help_pages/customformat.html), field formatting looks like this:
```
%[s][c][n.m]f[e]
```
Where `c` can be the optional encoding format for the field in question.  While usually one would want to use the same encoding across all fields specified in the output format by using the `%ZEncoding:STRING` option, there are cases where this needs to be done on a field by field basis.

As an example, if I wanted to output a plain URL to be handled by the latex `hyperref` package, I would want to avoid encoding its characters using the latex syntax.  In classic I can do this via this custom format:
```
%\1.1g \href{%u}{%\R}
```
This properly encodes the entry for 2013A&C.....1....1A as
```
Accomazzi A., et al.\ 
href{http://adsabs.harvard.edu/abs/2013A%26C.....1....1A}{2013A\&C.....1....1A} 
```
whereas in the new ADS I can only choose a global encoding and so the options I have are:
```
%1.1g \href{%u}{%R}
```
which gives
```
Accomazzi, and 7 colleagues \href{https://ui.adsabs.harvard.edu/\#abs/2013A&C...
..1....1A}{2013A&C.....1....1A}
```
and
```
%ZEncoding:latex %1.1g \href{%u}{%R}
```
which gives
```
Accomazzi, and 7 colleagues \href{https://ui.adsabs.harvard.edu/\textbackslash\#
abs/2013A\&C.....1....1A}{2013A\&C.....1....1A}
```

**Comment by golnazads** [comment]
At some point if we could please go over individual fields to determine which fields make sense to implement the encoding for https://docs.google.com/document/d/1Jn_ZJme0e7ass2cq6lJPVXVtsYAJyTwcBC2_GV4htFY/edit?usp=sharing

**Comment by golnazads** [comment]
field encoding was implemented in v2.0.39.
